src/datepicker/examples/bootstrap.datepicker.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.datepicker.componenet:pfBootstrapDatepicker
+ * @element pf-bootstrap-datepicker
+ *
+ * @param {date} date must be a Javascript Date - to be displayed in the input. Can be left empty.
+ * @param {string} format default date format for displayed dates
+ * @param {object} dateOptions an object to configure the uib-datepicker in one place (optional)
+ * @param {boolean} isOpen whether or not to show the datepicker (default false) (optional)
+ * @param {object} popupPlacement Placement of the datepicker popup relative to the input element. Passing in 'auto' separated by a space bfore the placement will enable auto positioning. (optional)
```

Comment:
Same as above vis a vis the object description, but additionally, with this option it might be nice to see the allowance for different placements in the demo area below. ie a radio button for say above/below.